src/pair.h

```diff
@@ -32,6 +32,8 @@ class Pair : protected Pointers {
   friend class Info;
 
  public:
+  enum{NONE,RLINEAR,RSQ,BMP};
```

Comment:
@sjplimp I think this warrants a more general discussion. I've opened #325 for that purpose.